src/main/java/Kevin.java

```diff
@@ -0,0 +1,150 @@
+                        System.out.println(line);
+                    } catch (IllegalCommandException e2) {
+                        System.out.println(line);
+                        System.out.println(e2.getMessage());
+                        System.out.println(line);
+                    }
+            }
+            userInput = scanner.nextLine();
+        }
+        scanner.close();
+    }
+
+    public static void printGreetMessage() {
+        String greet = line + "\n"
+                + "Hello! I'm Kevin.\n"
+                + "What can I do for you?\n"
+                + line;
+        System.out.println(greet);
+    }
+
+    public static void printByeMessage() {
+        String bye = line + "\n"
+                + "Bye. Hope to see you again soon!\n"
+                + line;
+        System.out.println(bye);
+    }
+
+    public static void listAllTasks() {
+        int count = 1;
+        System.out.println(line + "\n" + "Here are the tasks in your list:");
+        for (Task task: taskList) {
+            String string = String.format("%d.%s", count, task);
+            System.out.println(string);
+            count++;
+        }
+        System.out.println(line);
+    }
+
+    public static void markTaskDone(String taskNumber) {
+        Task currentTask = taskList.get(Integer.parseInt(taskNumber) - 1);
+        currentTask.setIsDone();
+        System.out.println(line + "\n");
+        System.out.println("Nice! I've marked this task as done:\n" + currentTask);
+        System.out.println(line);
+    }
+
+    public static void unmarkTaskDone(String taskNumber) {
+        Task currentTask = taskList.get(Integer.parseInt(taskNumber) - 1);
+        currentTask.setIsDone();
+        System.out.println(line + "\n");
+        System.out.println("Nice! I've marked this task as done:\n" + currentTask);
+        System.out.println(line);
+    }
+
+    public static void addNewTask(String userInput) throws DescriptionIncompleteException, IllegalCommandException{
+        String[] splitMessage = userInput.split(" ", 2);
+        String instruction = splitMessage[0];
+        Task task = null;
+
+        if (!(instruction.equals("todo")||instruction.equals("deadline")||instruction.equals("event"))) {
+            throw new IllegalCommandException("OOPS!!! I'm sorry, but I don't know what that means :-()");
+        }
+
+        if (splitMessage.length < 2) {
+            throw new DescriptionIncompleteException("OOPS!!! The description of an instruction cannot be empty.");
+        }
+
+        switch (instruction) {
+            case "todo":
+                task = Todo.createNewTodoTask(splitMessage[1]);
+                break;
+            case "deadline":
+                task = Deadline.createNewDeadlineTask(splitMessage[1]);
+                break;
+            case "Event":
+                task = Event.createNewEventTask(splitMessage[1]);
+                break;
+        }
```

Comment:
Problem same as above 🍔 
Is it because these code less line so you didn't put the curly braces this time? 😆